Bottom: edbddd22f9dcc0f354bf616adaaac8e02ed0adac
Top:    638426a6678afbd6c58621a75a306c8d0de5d84e
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-23 19:13:54 +0000

Remove PINFLAG, it is not required.

As nowhere cares that PINFLAG is set, we can avoid the code to handle this
and remove the flag.


---

diff --git a/src/create.c b/src/create.c
index 617fc4e..e1ab2a1 100644
--- a/src/create.c
+++ b/src/create.c
@@ -791,7 +791,6 @@ CreateNewPin (ElementType *Element,
   pin->Number = STRDUP (Number);
   pin->Flags = Flags;
   CLEAR_FLAG (WARNFLAG, pin);
-  SET_FLAG (PINFLAG, pin);
   pin->ID = ID++;
   pin->Element = Element;
 
diff --git a/src/strflags.c b/src/strflags.c
index d803e7e..8da69e2 100644
--- a/src/strflags.c
+++ b/src/strflags.c
@@ -486,9 +486,6 @@ common_flags_to_string (FlagType flags,
     case RATLINE_TYPE:
       CLEAR_FLAG (RATFLAG, &fh);
       break;
-    case PIN_TYPE:
-      CLEAR_FLAG (PINFLAG, &fh);
-      break;
     }
 #endif
